lib/nexpose/version.rb

```diff
@@ -1,4 +1,4 @@
 module Nexpose
   # The latest version of the Nexpose gem
-  VERSION = '3.1.0'
+  VERSION = '3.1.1'
```

Comment:
This would actually bump to version `3.2.0` since it's a backwards-compatible addition.